test/mason/masonInit/masonInitTest4.chpl

```diff
@@ -4,10 +4,12 @@ use MasonUtils;
 use MasonNew;
 
 proc main(){
-  const newArgs = ['mason','new','testSrc'];
+  const newArgs : [0..2] string;
+  const initArgs : [0..2] string;
+  newArgs = ['mason','new','testSrc'];
   masonNew(newArgs);
   
-  const initArgs = ['init','testSrc'];
+  initArgs = ['mason','init','testSrc'];
   masonInit(initArgs);
   
   runCommand('rm -rf testSrc');
```

Comment:
Mind updating the remaining `masonInit` tests to use `rmTree` over `runCommand`?